src/tools/Makefile.am

```diff
@@ -43,7 +43,8 @@ bin_PROGRAMS = \
 	pbs_wish \
 	printjob.bin \
 	printjob_svr.bin \
-	tracejob
+	tracejob \
+	pbs_sleep
```

Comment:
In windows this binary is named as `pbs-sleep` instead `pbs_sleep` so if you really want to create one sleep command for all platforms then I suggest to change this to `pbs-sleep` for non-windows platform.

Reply:
Makes sense that we should have the same name. However, PTL on windows is not yet available, so if we were to change the name in both platforms to pbs_sleep, that might be acceptable as well. Doesn't most of the PBS binaries on Windows have a underscore in the name as opposed to a dash?  If so, this is one chance to standardize on that as well.

Reply:
How many people do we think are using pbs-sleep on windows?  It would be bad if we suddenly changed the name of the process and broke anything a customer is doing.  I highly doubt anyone is using it except us for testing, but it's something to think about.  Do we need to deprecate pbs-sleep and add pbs_sleep for windows?  We can remove pbs-sleep one release later.

Reply:
That sounds like a great idea Bhroam. Perhaps we should deprecate pbs-sleep. I think we also missed asking the community about this. The slight laziness about this is probably based on the fact that we have not supported Windows from 14.2 onwards and are anyways making a huge bunch of changes to windows. PTL is not on windows either, so very few tests should get affected.

Another option is add a "soft-link" in windows (junction in windows parlance) to have a pbs-sleep that points to pbs_sleep. (as a method of having both and deprecating one..eventually removing the link in the future)